dubbo-rpc/dubbo-rpc-default/src/main/java/com/alibaba/dubbo/rpc/protocol/dubbo/telnet/ListTelnetHandler.java

```diff
@@ -61,10 +64,16 @@ public String telnet(Channel channel, String message) {
                 if (buf.length() > 0) {
                     buf.append("\r\n");
                 }
+                URL exportUrl = exporter.getInvoker().getUrl();
+                Map<String, String> params = StringUtils.parseQueryString(exportUrl.toFullString());

```

Comment:
You can use org.apache.dubbo.common.URL#getParameter(java.lang.String) instead of this.